source/includes/configuration/_02-get-historical-snapshot.md.erb

```diff
@@ -22,26 +28,23 @@ $ curl 'https://ci.example.com/go/api/admin/config/5059cf548db9ea2d1b9192b45529c
 </cruise>
 ```
 
-Gets the configuration with the specified version.
+Gets the current configuration file.
 
 <%= available_since('14.3.0') %>
 
 ### HTTP Request
 
-`GET /go/api/admin/config/:commit_sha.xml`
+`GET /go/api/admin/config/config.xml`
 
 **Other convenience APIs**
 
-`GET /go/api/admin/config/config.xml`
+`GET /go/api/admin/config/current.xml`
 
     or
 
-`GET /go/api/admin/config/current.xml`
+`GET /go/api/admin/config/:md5.xml`
```

Comment:
Can have a note about the headers (mentioned earlier) here. I wonder how useful this one is, any way. Without an API method to list MD5 sums of all (or some) of the config repo commits.